plugins/modules/clusters.py

```diff
@@ -80,10 +83,11 @@
 def run_module():
 
     module_args = dict(
-        action=dict(type="str", required=False, default="list"),
+        state=dict(type="str", required=False, default="list"),
```

Comment:
Add `choices=["absent", "present"]` to restrict inputs to valid values.

Reply:
Fixed in latest commit